sentry_kafka_management/actions/brokers.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any
+
+from confluent_kafka.admin import (  # type: ignore[import-untyped]
+    ConfigResource,
+)
+
+from sentry_kafka_management.brokers import ClusterConfig
+from sentry_kafka_management.connectors.admin import get_admin_client
+
+KAFKA_TIMEOUT = 5
+
+
+def describe_broker_configs(
+    kafka_config: ClusterConfig,
+) -> list[dict[str, Any]]:
```

Comment:
Consider returning `Sequence[Mapping[str, Any]]` So the implementation of the method can change the collection type without impacting the client.